__tests__/library.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {
+  toDoList, addToDo, removeList,
+} from '../src/modules/library.js';
+
+document.body.innerHTML = `
+  <section class="mainContainer">
+  <h1>Today's To Do</h1>
+  <div class="mainList">
+      <div class="upperOfList" >
+          <input class="addInput" id="addToDo" type="text" placeholder="Add your list ...">                
+      </div>
+      <ul id="detailedList"></ul>
+      <div>
+          <button type="button" class="clearAllBtn" id="clearAllBtn" > Clear all completed </button>
+      </div>
+  </div>
+  </section>     
+`;
+
+const eraseToDoList = () => toDoList.splice(0, toDoList.length);
+
+describe('Check add and remove', () => {
+  test('Adding an item', () => {
+    eraseToDoList();
+    addToDo('Wash the laundry');
+    addToDo('Travel to Arequipa');
+    expect(toDoList).toHaveLength(2);
+  });
+});
+
+test('Removing a task', () => {
+  eraseToDoList();
+  addToDo('Save the Queen');
+  addToDo('Jump to the sky');
+  addToDo('Go to the beach');
+  removeList(1);
+  expect(toDoList).toHaveLength(2);
+});
```

Comment:
- [ ] test should be grouped together using the `describe` block, however this test is outside of any block, please make sure to adjust this before requesting another review.